organization/_posts/2020/2020-11-05-coordination.md

```diff
@@ -0,0 +1,183 @@
+
+Was held [12 - 16 October](https://indico.cern.ch/event/946584/) with 50 participants. Lecture recordings are available on the Indico page. Very lively discussions and very good feedback from students. Many thanks again to Sebastien Ponce as lecturer and [all mentors](https://e-groups.cern.ch/e-groups/Egroup.do?egroupId=10387430).
+
+Another 180 people on a waiting list. In a post-mortem meeting we decided to go for another iteration with 75 participants max. Tentatively in the week of...
+
+**18 - 22 January 2021** --> *Please check if there are any clashes!!*
+
+(We will open registration anew, but let people on the waiting list know.)
+
+If you want to help us promoting training please consider using the info from this [propaganda slide](https://cernbox.cern.ch/index.php/s/s7V9FOVDejElBaD).
+
+Attrition rate was very low!
+
+Can improve the afternoon organisation for the students. Some of the rooms were quite quiet.
+
+Attila - should we also discuss higher level concepts (singletons, factory pattern)? Liz - singletons are very concurrency unfriendly though.
+
+- There will be a later meeting to discuss the curriculum.
+
+### Compute Accelerator Forum
+
+Next meeting, [Wed 11 Nov, 17:00 CET](https://indico.cern.ch/event/962110/) with talks given by David Rohr (ALICE) and Daniel Campora (LHCb) on their experiments' approaches to abstraction of GPU hardware.
+
+Currently discussing with NVidia the possibility to have a seminar style presentation on either the NSight profiling suite or libcu++ for the [Wed 9 Dec, 16:30 CET](https://indico.cern.ch/event/962112/) meeting.
+
+Tentative dates and times for 2021 -- 2nd Wednesday of the month at 16:30 -- have been entered in the [indico category](https://indico.cern.ch/category/12741/).
+
+Please subscribe to [compute-accelerator-forum-announce@cern.ch](https://e-groups.cern.ch/e-groups/EgroupsSubscription.do?egroupName=compute-accelerator-forum-announce) to receive notifications.
+
+## LHCC
+
+### WLCG Software Liasons
+
+The WLCG has appointed Graeme Stewart and Liz Sexton-Kennedy as their software liason contacts. Graeme and Liz will take care of reporting for software groups and for communicating WLCG needs back to developers.
+
+- No intention for the current lines of communication between experiments and projects to be cut, but WLCG is an all-experiment forum that can be useful for setting priorities as seen by the WLCG
+- Generally we want to improve communications between computing and software, e.g., organise a one-shot workshop on I/O matters
+
+### Referees Meeting
+
+There is an LHCC Referees Meeting with WLCG on Tuesday 17 November. Graeme will give a (general) software update (20' slot). Please send any input that you have.
+
+### HL-LHC Software and Computing Review
+
+We also expect to get news about the charge from the LHCC about the next phase of the review of software and computing for HL-LHC, this time focussing on common software projects, that should take place in Autumn 2021.
```

Comment:
focussing => focusing

Reply:
Spelling with a "ss" is allowed (the OED says so) 🤣